### PR TITLE
Changes to DPI api

### PR DESCRIPTION
**Pull request by Tbaile**
- renamed DPI parameter `interface` to `device`, removing every doubt that would have come up regards of what this field was supposed to contain
- added `list-devices` api and examples

Requiring `python3-nethsec:0.0.8` to work, please wait for NethServer/python3-nethsec#13 to be merged and tagged.

### CODE REVIEW

**Review by gsanchietti — APPROVED** [review]
Remember to bump also ns-api after the merge

**Comment by Tbaile** [comment]
Better bumping now the version, cannot merge this PR myself :shrug:

**Pull request merged**
